esmvaltool/diag_scripts/shared/plot/xy_line.ncl

```diff
@@ -1520,45 +1520,45 @@ begin
 
   ; Select colors and other plotting attributes for multi-model mean
   if (diag_script_info@multi_model_mean .eq. "y") then
-    ; Project_style evaluates metadata of variable "models"
-    temp = models
+    ; Project_style evaluates metadata of variable "dataset_info"
```

Comment:
This block is probably no longer required, as long as the multi-model mean is done by the preprocessor.